packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -102,9 +102,14 @@ export async function createNewFluidFile(
     const odspUrl = createOdspUrl({... newFileInfo, itemId, dataStorePath: "/"});
     const resolver = new OdspDriverUrlResolver();
 
-    // caching the converted summary
+    // converting summary and getting sequence number
     const snapshot: IOdspSnapshot = convertSummaryTreeToIOdspSnapshot(createNewSummary);
-    const value: ISnapshotCacheValue = { snapshot, sequenceNumber: 0 };
+    const protocolSummary = createNewSummary.tree[".protocol"] as ISummaryTree;
+    const documentAttributes = getDocAttributesFromProtocolSummary(protocolSummary);
+    const sequenceNumber = documentAttributes.sequenceNumber;
```

Comment:
I glanced around and it turned out it's much simpler - we do have sequence number as containerSnapshot.sequenceNumber - take a look at convertSummaryIntoContainerSnapshot(), it does the same thing, so no need to duplicate it here :)